src/extension/provider/statefulAuth.ts

```diff
@@ -248,6 +237,71 @@ export class StatefulAuthProvider implements AuthenticationProvider, Disposable
     this.#disposables.forEach((d) => d.dispose())
   }
 
+  public async loadFromSecrets() {
```

Comment:
It's a `token`. More specifically, "secrets" is too vague.

Also, perhaps it should be `loginFromToken` or `bootstrapFromToken` because it does not "just" do loading.